service/frontend/adminHandler_test.go

```diff
@@ -764,3 +776,207 @@ func (s *adminHandlerSuite) Test_GetDynamicConfig_FilterMatch() {
 	s.NoError(err)
 	s.Equal(resp.Value.Data, encTrue)
 }
+
+func Test_GetGlobalIsolationGroups(t *testing.T) {
```

Comment:
the test is more like an integration test
I think we should mock `IsolationGroupState` instead of mocking its dependencies.